landscape/explanation/glossary/ppa.md

```diff
@@ -0,0 +1,6 @@
+# PPA 
+
+**Personal Package Archive(PPA)** is a software repository specifically made for Ubuntu-based products. This tool allows developers to create their own repositories that they can use to distribute their own software. In the context of Landscape, the PPA contains the tool's software for the server. 
```

Comment:
```suggestion
A **Personal Package Archive (PPA)** is a software repository specifically made for Ubuntu users. PPAs are used by developers to distribute their software. When you install Landscape, you provide a PPA so your system knows where to download Landscape and its required packages. For example, if you want the beta version of Landscape, you must add the Landscape beta PPA to your system.
```

Reply:
Note: I did minimize the second sentence here a bit - your information wasn't wrong, but I just felt it might be confusing for Landscape users if we described PPAs in such a general sense. You are correct that developers can create their own repos and distribute their own software with PPAs, but since Landscape uses specific PPAs owned by Canonical, I changed that part to be more broad.